doc/install-tizen-sdk.md

```diff
@@ -11,18 +11,18 @@ After installing Tizen Studio, the _Package Manager_ window will automatically p
 
 ![Tizen Package Manager](images/package-manager.png)
 
-- **Required**
-  - _[Tizen SDK tools] - [Native CLI]_
-  - _[Tizen SDK tools] - [Baseline SDK] - [Certificate Manager]_
-  - _[Extension SDK] - [IOT-Headed-6.0] - [Advanced] - [Native app. development (CLI)]_
-  - _[Extension SDK] - [Samsung Certificate Extension]_
+- [Tizen SDK tools] - [Native CLI]
+- [Tizen SDK tools] - [Baseline SDK] - [Certificate Manager]
+- [Extension SDK] - [IOT-Headed-6.0] - [Native app. development (IDE)]
```

Comment:
Are there any other tools(?) required beyond the CLI?

Reply:
No, as far as I know.

GCC 9.2 toolchain should be automatically installed as a dependency.